flow_screen_components/richTextInputFSC/force-app/main/default/lwc/inputRichTextFSC/inputRichTextFSC.html

```diff
@@ -1,5 +1,58 @@
 <template>
-    <lightning-input-rich-text
-        value={value} formats={formats} label={label} label-visible=true>
-    </lightning-input-rich-text>
+
+    <!-- Start Rich Text Input Section - if richerText is false - this is default -->
+    <template if:false={richerText}>
+        <div>
+            <lightning-input-rich-text value={value} formats={formats} label={label} label-visible=true></lightning-input-rich-text>
+        </div>
+    </template>
+    <!-- End Rich Text Input Section-->
+
+    <!-- Start Entry Enhancements - only display if richerText activated -->
+    <template if:true={richerText}>
+        <div>
+            <lightning-input-rich-text value={richText} formats={formats} label={label} onchange={handleTextChange} label-visible=true>
```

Comment:
per DRY principles, we should only have a single lightning-inpuyt-rich text. Can you unify the call , and make just the bottom toolbar subject to the conditionality?

Reply:
Here I am having an issue with value attribute when entering search or replace text.  It is entering into the onchange event for the value text field for some reason, and cannot figure out why.  Thought this split would be OK, and to keep the event off for the existing section using value.

Reply:
That's probably worth seeking an answer on stack exchange, but feel free to leave it the way it is.